web/components/admin/Toolbar/index.js

```diff
@@ -22,6 +29,13 @@ class Toolbar extends React.Component {
           name="collection-input"
           value={this.collectionSelectLabels[applicantType]}
           options={Object.values(this.collectionSelectLabels)} />
+        <Select
+          onChange={option => switchSortType(option.value)}
+          label="Sort:"
+          name="sort-input"
+          value={this.sortSelectLabels[sortType]}
+          options={Object.values(this.sortSelectLabels)} />
+        <button onClick={reverseSort}>{!reverse ? 'reverse' : 'unreverse'}</button>
```

Comment:
This works fine but if you want to to look super good, we already have an icon on Figma for ascending/descending order. You can export it and try to replace the button with it (it can just be a regular `<img>` with an `onClick`).
![image](https://user-images.githubusercontent.com/12876696/49784151-2c83ce00-fcd1-11e8-93e9-8dfff9772671.png)
To flip it upside down, we have an existing utility class (`.flip-vertical`) in [animations.sass](https://github.com/nwplus/nwhacks2019/blob/master/web/styles/animations.sass#L55-L57) that you can add to the `<img>` conditionally.

Reply:
Very cool! 

I was wondering where to find those graphics.

Reply:
^ If that last part is a question, you can find it on the Internal page:
![image](https://user-images.githubusercontent.com/12876696/49787442-91442600-fcdb-11e8-8e53-b667d12fc5a0.png)
<details>
<summary>To export, you can click on the icon and then on the bottom of the panel that appears on the right you should be able to export it as an SVG file
</summary>
<img src="https://user-images.githubusercontent.com/12876696/49787536-cea8b380-fcdb-11e8-8b8f-8cf008aa92a0.png" />
</details>

Reply:
Flip vertical applies an animation when its added, but not when its removed.

Any way to get around this?

Reply:
Oh true, you can fix that by adding a transition to `.sort-arrow`:
```
  .sortArrow
    @include transition(0.2s)
    ....
```
This delay will apply to any value changes of [all animatable properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties), but we're only changing the `transform` property for the arrow so it should be fine.